setup.py

```diff
@@ -5,8 +5,8 @@
 from torch.utils.cpp_extension import BuildExtension, CUDAExtension
 
 FASTSEQ_VERSION = '0.0.4'
-MIN_FAIRSEQ_VERSION = '0.9.0'
-MAX_FAIRSEQ_VERSION = '0.9.0'
+MIN_FAIRSEQ_VERSION = '0.10.2'
+MAX_FAIRSEQ_VERSION = '0.10.2'
```

Comment:
Are these changes compatible with any 0.10.x?

Reply:
Yes. updated the min fairseq version